scripts/update-external.nix

```diff
@@ -1,5 +1,5 @@
 { stdenv, writeScript, glibc, coreutils, git, openssh
-, nix-tools, cabal-install, nix-prefetch-git
+, internal-nix-tools, internal-cabal-install, nix-prefetch-git
```

Comment:
Other end of my nit above. I guess my reasoning is that this Nix file "doesn't care" about the nix-tools/cabal-install, it's the caller that decides it's going to be the internal one.

And likewise below.